packages/utils/src/index.js

```diff
@@ -1,4 +1,5 @@
-import { useRef, useMemo, useEffect } from "react";
+import { useRef, useMemo, useEffect, useCallback } from "react";
+import useLatest from "use-latest";
```

Comment:
If you want - I can inline this dependency. It's really simple but my personal opinion is that it's good to share common code (even if it's short like this one), but I know many people prefer inlining for cases like this. Disclaimer though - it's written by me 😉 I actually also probably will extract `useStableCallback` to a separate library as well, just didn't have time to do this right now.